ui/shared/components/panel/family/FamilyReads.jsx

```diff
@@ -189,7 +204,9 @@ const IgvPanel = React.memo((
     size,
   )
 
-  const tracks = rnaReferences.concat(getIgvTracks(igvSampleIndividuals, individualsByGuid, sampleTypes))
+  const tracks = applyUserTrackSettings(
+    rnaReferences.concat(getIgvTracks(igvSampleIndividuals, individualsByGuid, sampleTypes)), options,
+  )
```

Comment:
As I say below, we should be passing `minJunctionEndsVisible`, and I think we don't wan to get too bogged down in making this generic given that we don't exactly know what  future enhancements would be. That said, I think we would want to apply all options for a track type at once, not iterate through multiple times if we end up needing to add multiple properties to the same track type. I think we should replace `options` here with 
`{ [JUNCTION_TYPE]: { minJunctionEndsVisible } }` and then update `applyUserTrackSettings` accordingly